src/main/java/gregtech/api/metatileentity/interfaces/INeighborCache.java

```diff
@@ -0,0 +1,15 @@
+package gregtech.api.metatileentity.interfaces;
+
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public interface INeighborCache extends IHasWorldObjectAndCoords {
```

Comment:
What is the point of this interface?

Reply:
This is just to group together common functionality between GT tile entities. It also means we can have javadocs unified across all implementations of the iface.